docs/developers/dapp-developer/native.md

```diff
@@ -86,17 +86,24 @@ nonce, err := cl.NonceAt(context.Background(), addr, big.NewInt(14000000))
 
 ### Querying past events {#querying-past-events}
 
-Contracts emit events during execution which can be queried from the client. The parameters for the event one is interested in have to be filled out in the `ethereum.FilterQuery` object. This includes which event topics are of interested, from which contracts and during which range of blocks. The example below queries `Transfer` events of all ERC-20 tokens for the last 10 blocks:
+Contracts emit events during execution which can be queried from the client. The parameters for the event one is interested in have to be filled out in the `ethereum.FilterQuery` object. This includes which event topics are of interest, from which contracts and during which range of blocks. The example below queries `Transfer` events of all ERC-20 tokens for the last 10 blocks:
 
 ```go
 blockNum, err := cl.BlockNumber(context.Background())
 if err != nil {
     return err
 }
+
+transferEventSignature := []byte("Transfer(address,address,uint256)")
+transferEventHash := crypto.Keccak256Hash(transferEventSignature)
+transferEventHashHexString = transferEventHash.Hex() // "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
+
 q := ethereum.FilterQuery{
     FromBlock: new(big.Int).Sub(blockNum, big.NewInt(10)),
     ToBlock: blockNum,
-    Topics: [][]common.Hash{common.HexToHash("0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef")},
+    Topics: [][]common.Hash{
+		  []common.Hash{common.HexToHash(transferEventHashHexString)}
+		},
```

Comment:
```suggestion
    Topics: [][]common.Hash{
		  []common.Hash{transferEventHash}
		},
```

Reply:
@holiman 1000% agree thank you.  I've also updated for the typecasting of the blocknum value from uint to into to big.

Realised the old code wasnt compiling for me.

Reply:
@holiman  any chance you've got time to take a quick look? Other PRs to come soon as the docs are a bit outdated and I'm working on some training material for those new to geth so this is my way of learning to teach :)